src/Tribe/Admin/Notice/Marketing/Stellar_Sale.php

```diff
@@ -66,10 +66,19 @@ public function display_notice() {
 		$cta_url      = 'https://evnt.is/1bcv';
 		$stellar_url  = 'https://evnt.is/1bcu';
 		$end_date     = $this->get_end_time();
+		$template_args = [
+			'cta_url'      => 'https://evnt.is/1bcv',
+			'stellar_url'  => 'https://evnt.is/1bcu',
+			'end_date'     => $this->get_end_time(),
+		];
+		$dependency   = tribe( \Tribe__Dependency::class );
 
 		ob_start();
-
-		include \Tribe__Main::instance()->plugin_path . 'src/admin-views/notices/tribe-stellar-sale.php';
+		if ( $dependency->has_active_premium_plugin() ) {
+			$this->get_template()->template( 'notices/tribe-stellar-sale-premium', $template_args, true );
+		} else {
+			$this->get_template()->template( 'notices/tribe-stellar-sale', $template_args, true );
+		}
```

Comment:
Remove the `ob_*` methods here in favor of returning directly from template.

```suggestion
		if ( $dependency->has_active_premium_plugin() ) {
			return $this->get_template()->template( 'notices/tribe-stellar-sale-premium', $template_args, false );
		} else {
			return $this->get_template()->template( 'notices/tribe-stellar-sale', $template_args, false );
		}
```